front/src/applications/stdcm/hooks/useStdcmConsist.ts

```diff
@@ -58,7 +61,12 @@ const useStdcmConsist = () => {
     }
 
     if (!maxSpeedChanged) {
-      const consistMaxSpeed = min([rollingStock?.max_speed, towed?.max_speed]);
+      const maxSpeedFromTag = maxSpeedFromSpeedLimitByTag(maxSpeedTag);
```

Comment:
I think that's not the expected behavior, there is this mention in the issue : `if the chosen code is supposed to lower it.`. So for example, switching from MA80 to MA120 shouldn't affect the vmax.
Is it what you meant @axrolld ?

Reply:
Ok behavior seems perfect !

Reply:
Just wondering, on my computer the tower rollingstock endpoint returns an empty array. Same for you ?

Reply:
No, on my computer, the towed endpoint returns data.